packages/react/src/ReactServer.fb.js

```diff
@@ -37,6 +39,12 @@ const Children = {
   only,
 };
 
+// Only export captureOwnerStack if the flag is on, to support feature detection.
+let captureOwnerStack: ?() => null | string;
+if (enableOwnerStacks) {
+  captureOwnerStack = captureOwnerStackImpl;
+}
```

Comment:
This is sus, why does this need exported here but not from the other `ReactServer.js` exports?